transformer/cloudfoundrytransformer.go

```diff
@@ -263,21 +299,43 @@ func (t *CloudFoundry) prioritizeAndAddEnvironmentVariables(cfApp collecttypes.C
 			flattenedEnvList := flattenVcapServiceVariables(valueStr)
 			for _, env := range flattenedEnvList {
 				envOrderMap[env.Name] = env
+				vcapEnvMap[env.Name] = []byte(env.Value)
 			}
 		}
+		dbEnv, err := createDbEnvironmentVariable(valueStr, serviceName)
+		if err != nil {
+			logrus.Errorf("Could not create DATABASE_URL environment: %v", err)
+		} else {
+			envOrderMap[dbEnv.Name] = dbEnv
+			vcapEnvMap[dbEnv.Name] = []byte(dbEnv.Value)
+		}
 		envOrderMap[varname] = core.EnvVar{Name: varname, Value: valueStr}
+		vcapEnvMap[varname] = []byte(valueStr)
 	}
 	for varname, value := range cfApp.Environment.ApplicationEnv {
-		envOrderMap[varname] = core.EnvVar{Name: varname, Value: fmt.Sprintf("%s", value)}
+		valueStr := fmt.Sprintf("%s", value)
```

Comment:
Why not call normalizeForEnvironmentVariableName function directly here?

Reply:
we strictly dont need normalizeForEnvironmentVariableName() in this location as we are not unpacking VCAP_APPLICATION variable.